bundles/net.tourbook/src/net/tourbook/map3/layer/tourtrack/TrackPathOptimized.java

```diff
@@ -572,12 +570,12 @@ protected void doDrawOutlineVBO(final DrawContext dc, final int[] vboIds, final
          }
 
          if (!isPickingMode && config.isShowDirectionArrows) {
-            drawDirectionArrows(dc, vboIds, pathData);
+            drawDirectionArrows(dc, vboIds);
          }
 
       } finally {
-         gl.glBindBuffer(GL2.GL_ARRAY_BUFFER, 0);
-         gl.glBindBuffer(GL2.GL_ELEMENT_ARRAY_BUFFER, 0);
+         gl.glBindBuffer(GL.GL_ARRAY_BUFFER, 0);
+         gl.glBindBuffer(GL.GL_ELEMENT_ARRAY_BUFFER, 0);

```

Comment:
Why are many GL* constants changed into GL ?

This is a very complex code (at least for me) and should not be modified when not needed

Reply:
> Maybe an Eclipse optimization ?

I don't now how you changed it, there are also conversions from `4` to `4L`, this seems also to be automated